hw/dv/data/riviera/riviera.hjson

```diff
@@ -0,0 +1,84 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  build_cmd:  "{job_prefix} vlog"
+  run_cmd:    "{job_prefix} vsim"
+
+  // Indicate the tool specific helper sources - these are copied over to the
+  // {tool_srcs_dir} before running the simulation.
+  tool_srcs:  ["{proj_root}/dv/uvm/data/riviera/*"]
+
+  build_opts: ["-sv",
+               "-timescale 1ns/1ps",
+               "+incdir+$$ALDEC_PATH/vlib/uvm-1.2/src",
```

Comment:
```suggestion
               "+incdir+{ALDEC_PATH}/vlib/uvm-1.2/src",
```
If it is all the same to you, can this be `ALDEC_HOME` instead (to match with other simulators)?